docs/build/identity.rs/1.1/docs/how-tos/verifiable-credentials/revocation.mdx

```diff
@@ -0,0 +1,200 @@
+---
+sidebar_label: Revoke
+description: Explain how a VC can be revoked
+image: /img/Identity_icon.png
+keywords:
+  - verifiable
+  - credentials
+  - revoke
+  - revocation
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Revoke a Verifiable Credential
+
+The [example](#example) below demonstrates two methods that an issuer can use to revoke a verifiable credential
+using the IOTA Identity Framework:
+
+1. By using the [`credentialStatus`](https://www.w3.org/TR/vc-data-model/#status) field in a credential and linking
+to a [revocation method](#revocation-methods).
+2. By [removing the verification method](#removing-the-verification-method) that signed the credential.
+This invalidates all credentials that were signed with that verification method.
+
+## Revocation methods
+The IOTA Identity Framework supports two different revocation methods: `RevocationBitmap2022` and `StatusList2021`. 
+### Revocation Bitmap
+[RevocationBitmap2022](/identity.rs/references/specifications/revocation-bitmap-2022/) is the default credential revocation method used in the IOTA Identity Framework. It allows
+issuers to control whether a credential is _valid_ or _revoked_. To do so, a revocation list (represented
+as a bitmap) is stored in the issuer's DID document.
```

Comment:
Can we maybe add an info box with size constraint/rough number of possible revocations here?